src/components/PlayerList/styles.styl

```diff
@@ -53,6 +53,8 @@
   width: 45px
   height: 45px
   border-radius: 3px
+  transition: all 300ms ease-in-out
+  background-size: contain
   background: #e2e2e2
```

Comment:
this needs to be `background-color` because it unsets the `background-size`. See https://deploy-preview-3--friendgame-report.netlify.com/ps/thisjoshthat

Reply:
honestly, it blows my mind that background image can be animated like this. TIL 💯

Reply:
Oops, should have checked that after the merge.